### PR TITLE
Update omnipresence to 1.5.1

### PR DESCRIPTION
**Pull request by colindunn**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by colindunn** [comment]
Hoping my definition of versioning in the path makes sense. Could just use MacOS.version except that both 10.11 and 10.12 are usable on the latest version which doesn't match the others. Anyway, let me know if there is a better way here

File: Casks/omnipresence.rb [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-  sha256 '409bf272e7c4dc488f68abadb3e2ef15d4accde10f8ee9babd8b23f522bfe323'
+  if MacOS.version == :mavericks
+    version '1.2-r220586'
+    sha256 '59a214e6393c984aead4a7242297979dc553d59af9964b5c645636e395794f0f'
+    osx_version_path = MacOS.version
+  elsif MacOS.version == :yosemite
+    version '1.4.1'
+    sha256 '409bf272e7c4dc488f68abadb3e2ef15d4accde10f8ee9babd8b23f522bfe323'
+    osx_version_path = MacOS.version
+  else # MacOS.version >= :el_capitan
```
Comment (vitorgalvao) [inline Casks/omnipresence.rb]: Remove the comment.

File: Casks/omnipresence.rb [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+    version '1.5.1'
+    sha256 'e785b910e7413986be45ab14f6a86dc598999461bf3c2a9d0da3759848342ac7'
+    osx_version_path = '10.11'
+  end
 
-  url "https://downloads.omnigroup.com/software/MacOSX/10.10/OmniPresence-#{version}.dmg"
+  url "https://downloads.omnigroup.com/software/MacOSX/#{osx_version_path}/OmniPresence-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/omnipresence.rb]: Yes, use `MacOS.version` instead.

File: Casks/omnipresence.rb [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
+    sha256 '59a214e6393c984aead4a7242297979dc553d59af9964b5c645636e395794f0f'
+    osx_version_path = MacOS.version
+  elsif MacOS.version == :yosemite
+    version '1.4.1'
+    sha256 '409bf272e7c4dc488f68abadb3e2ef15d4accde10f8ee9babd8b23f522bfe323'
+    osx_version_path = MacOS.version
+  else # MacOS.version >= :el_capitan
+    version '1.5.1'
+    sha256 'e785b910e7413986be45ab14f6a86dc598999461bf3c2a9d0da3759848342ac7'
+    osx_version_path = '10.11'
```
Comment (vitorgalvao) [inline Casks/omnipresence.rb]: If `10.12` also works, no point in freezing this one, then.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by colindunn** [comment]
The links are as follows:

mavericks: https://downloads.omnigroup.com/software/MacOSX/10.9/OmniPresence-1.2-r220586.dmg
yosemite: https://downloads.omnigroup.com/software/MacOSX/10.10/OmniPresence-1.4.1.dmg
el_capitan: https://downloads.omnigroup.com/software/MacOSX/10.11/OmniPresence-1.5.1.dmg
sierra: https://downloads.omnigroup.com/software/MacOSX/10.11/OmniPresence-1.5.1.dmg

Sierra doesn't use 10.12, which was why I didn't use MacOS.version but used a variable instead, trying to avoid using url in each if block.

Would you prefer me to just use url in each instead of doing it this way?

File: Casks/omnipresence.rb (hunk 2 above)
Comment (colindunn) [inline Casks/omnipresence.rb]: See below

File: Casks/omnipresence.rb (hunk 3 above)
Comment (colindunn) [inline Casks/omnipresence.rb]: See below

File: Casks/omnipresence.rb (hunk 1 above)
Comment (colindunn) [inline Casks/omnipresence.rb]: Will do

**Comment by vitorgalvao** [comment]
Using the variable in some and setting one of them to a fixed value seems inelegant. We don’t really gain anything by using `MacOS.version` in the early versions (which will not receive more updates and hence not change) if for the later versions we need a fixed value.

I’d rather use the `*_comma` methods. So we’d have `version '1.2-r220586,10.9'` and so forth. The URL would use `"https://downloads.omnigroup.com/software/MacOSX/#{version.after_comma}/OmniPresence-#{version.before_comma}.dmg"`.

**Comment by colindunn** [comment]
Done

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**